### DISCUSSION TITLE
Custom MultiCommands (with styling)

### DISCUSSION THREAD

**Discussion by wabiloo** [janluke/cloup#165] [Q&A]
I am trying to enable a plugin-based architecture, along the lines of the click documentation at https://click.palletsprojects.com/en/8.1.x/commands/#custom-multi-commands.

However, when I do that, naturally the output looks black and white, and not the cloup colored output I've got used to.

I was thinking I could just replace the inheritance to `cloup.MultiCommand`, but that does not seem to exist.

Am I missing something obvious?

**Comment by janluke** [comment]
Try extending `cloup.Group` or take inspiration from the the following branch and discussion: 
- https://github.com/janluke/cloup/compare/feat/lazy-loading
- https://github.com/janluke/cloup/discussions/102

**Comment by janluke** [comment]
Have you solved your problem?

**Comment by wabiloo** [comment]
I tried inheriting from cloud.Group instead of click.MultiCommand, but that didn't work (the sub-commands did not get listed). 
I need to review your lazy loading example

**Comment by janluke** [comment]
@wabiloo If you have any feedback on lazy-loading, please share it here: https://github.com/janluke/cloup/issues/103.

Note that click maintainers are considering deprecating `MultiCommand`.

**Reply by janluke** [reply]
Update: `MultiCommand` is now deprecated.
